src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -141,6 +141,13 @@ internal override void ProcessResponse(HttpResponseMessage response)
             }
             else if (ShouldSaveToOutFile)
             {
+                // Check if OutFile is a folder
+                if (Directory.Exists(OutFile))
```

Comment:
`OutFile` can be provider-qualified path or relative path, ex. `FileSystem::~\out.zip`.
I think we should use `QualifiedOutFile`.

Reply:
I changed it, but it was working correctly with `FileSystem::~\out.zip` and `FileSystem::~\out`